src/status_im/chat/models/delete_message_for_me_test.cljs

```diff
@@ -0,0 +1,99 @@
+(ns status-im.chat.models.delete-message-for-me-test
+  (:require
+   [cljs.test :refer-macros [deftest is testing]]
+   [status-im.chat.models.delete-message-for-me :as to-test]))
+
+(defonce mid "message-id")
+(defonce cid "chat-id")
+
+(deftest delete-for-me
+  (let [db      {:messages {cid {mid {:id mid}}}}
+        message {:message-id mid :chat-id cid}]
+    (testing
+     "delete for me"
```

Comment:
[Code style] I think it's more common to leave the test description in the same line as the `testing` macro, both in status-mobile codebase and in the wild.

Reply:
I agree, this is more friendly to grep. 
I found zprint want to put it into a newline. Let me check if there's anyway to tweak zprint to respect the same line in `testing` form

Reply:
I see, you're already using zprint. I think it gets confused with the line length. It shouldn't break the line for certain macros like this one, which expect a description that will often go past 80 characters long.